packages/backend/src/db/bill/bill.schema.ts

```diff
@@ -0,0 +1,46 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document, Types } from 'mongoose';
+
+export type BillDocument = Bill & Document;
+
+@Schema()
+export class Bill {
+  @Prop()
+  name: string;
+
+  @Prop()
+  description: string;
+
+  @Prop({ type: Types.ObjectId, ref: 'House' })
+  house: Types.ObjectId;
+
+  @Prop({ type: Types.ObjectId, ref: 'User' })
+  owner: Types.ObjectId;
```

Comment:
By the looks of the issue specification this should also be the users firebaseId, and therefore a string

Reply:
I think the ````"owner": "FirebaseID"```` in the issue is for Bill-Response-Dto,which they have defined and make it to be firebaseId in the controller so i guess should be fine. Same idea has been used for houseSchema and house-response dto.